internal/backend/runtime/omni/state_validation.go

```diff
@@ -1120,3 +1120,20 @@ func infraMachineConfigValidationOptions(st state.State) []validated.StateOption
 		})),
 	}
 }
+
+func nodeForceDestroyRequestValidationOptions(st state.State) []validated.StateOption {
```

Comment:
We probably need the unit test for that.

Reply:
good point, added